stylix/autoload.nix

```diff
@@ -9,6 +9,11 @@ builtins.concatLists (
     let
       file = "${inputs.self}/modules/${path}/${for}.nix";
```

Comment:
This is a good place to put `isFunction` and `expectsMkTarget` variables :wink: 

```nix
let
  file = "${inputs.self}/modules/${path}/${for}.nix";
  module = import file;
  # Assume that a function-module whose first arg doesn't have any de-structured attrs is actually expecting `mkTarget` as its first arg
  expectsMkTarget = builtins.isFunction module && builtins.functionArgs module == { };
in
```

The comment & named var are intended to make it clear _why_ we care whether the function-args has any declared de-structured attrs.

Reply:
Maybe also worth commenting _why_ we supply the function in this way at all? Someone reading the code will likely wonder why we can't just put it in `config.lib.stylix`. with everything else.